primefaces/src/main/resources/META-INF/primefaces-p.taglib.xml

```diff
@@ -131,25 +131,25 @@
             org.primefaces.component.export.DataExporterTagHandler
         </handler-class>
         <attribute>
-            <description>Server side id of the datatable whose data would be exported.</description>
+            <description>Search expression to resolve one or multiple datatables.</description>
             <name>target</name>
             <required>true</required>
             <type>java.lang.String</type>
         </attribute>
         <attribute>
-            <description>Export type: "xls","pdf","csv", "xml".</description>
+            <description>Export type: "xls", "xlsx", "xlsxstream", "pdf", "csv", "xml"</description>
             <name>type</name>
             <required>true</required>
             <type>java.lang.String</type>
         </attribute>
         <attribute>
-            <description>Filename of the generated export file, defaults to datatable server side id.</description>
+            <description>Filename of the generated export file, defaults to datatable id.</description>
             <name>fileName</name>
             <required>false</required>
             <type>java.lang.String</type>
         </attribute>
         <attribute>
-            <description>Exports only current page instead of whole dataset.</description>
+            <description>Exports only current page instead of whole dataset</description>
```

Comment:
```suggestion
            <description>Exports only current page instead of whole dataset.</description>
```

Reply:
@jepsar how did you do this where its a "suggested" change? I have never seen this before and I am trying to figure out how you did it?

Reply:
I tried to add the removed period again.

Reply:
Oh, sorry, do you mean in GitHub? Just click the add suggestion button.

<img width="229" alt="Screenshot 2022-07-01 at 14 53 16" src="https://user-images.githubusercontent.com/7500178/176898427-1f66cce1-b489-40c4-b004-b63d89522199.png">

Reply:
Awesome I never noticed that button before!